db/migrations/20180102183100_add_stack_to_buildpack_table.rb

```diff
@@ -0,0 +1,19 @@
+require 'yaml'
+
+Sequel.migration do
+  up do
+    alter_table(:buildpacks) do
+      add_column :stack, String, size: 255, null: true
+      drop_index :name, unique: true
```

Comment:
According to the [Sequel docs](https://sequel.jeremyevans.net/rdoc/classes/Sequel/Schema/CreateTableGenerator.html#method-i-index), Sequel will attempt to use the same name it would have generated when adding the index initially. This should be fine for most cases. 

The naming logic looks like this:
https://github.com/jeremyevans/sequel/blob/7c7d14d00a2cfd3ccb7bfef7fc6f050a12e264dc/lib/sequel/database/schema_methods.rb#L775

I'm a bit worried that this will result in failed upgrades for some users, but hopefully that's just FUD since I can't think of a way around doing this.

Reply:
Weird. I checked the database before migrating and `SHOW INDEX on buildpacks` shows the name was `buildpacks_name_index`, but changing the migration to `drop_index name: :buildpacks_name_index` yields this error: 
`Sequel::DatabaseError: PG::UndefinedObject: ERROR:  index "buildpacks___name____buildpacks_name_index___index" does not exist`

So I have reverted to not passing the name here, but I'm happy to try other suggestions; our team is not experienced with Sequel.